scripts/regression_test/extension_regression_test.yml

```diff
@@ -128,18 +128,18 @@ jobs:
   - task: PublishTestResults@2
     condition: succeededOrFailed()
     inputs:
-      testResultsFiles: '~/.azdev/env_config/mnt/vss/_work/1/s/env/test_results_*.xml'
+      testResultsFiles: '/$(HOME)/.azdev/env_config/mnt/vss/_work/1/s/env/test_results_*.xml'
```

Comment:
The original path will fail with
```
Not found PathtoPublish: /mnt/vss/_work/1/s/~/.azdev/env_config/mnt/vss/_work/...
```

Have to add `/` at the beginning to avoid auto complete `mnt/vss/_work/1/s`, also use `$(HOME)` to replace `~`